README.md

```diff
@@ -334,8 +334,6 @@ The hard reality to keep in mind is that you shall always have an interface lite
 If such an interface does not exist after CNI is invoked (also having an IPv4 address), the state of the Pod will be considered "faulty", and it will be re-created in a loop.
 To be able to comply with this Kubernetes limitation, DANM always names the first container interface "eth0", regardless of your intention.
 
-Sorry, but they made us do it :)
```

Comment:
I'm generally okay with grammar, phrasing, restructuring etc. related corrections but at the same time I would not like the project README to become an official CuDo when it comes to writing style.
being lighthearted is part of the appeal, and the style-feedback from the readers was mostly positive so far

so please don't delete such remarks (assuming it was done purely because of style)

Reply:
Hi, 
OK, I'll put it back then. :)